NEWS.md

```diff
@@ -4,6 +4,12 @@ ClimaLand.jl Release Notes
 main
 --------
 
+v0.11.3
```

Comment:
https://github.com/CliMA/ClimaLand.jl/pull/575 was breaking, so the next release will have to be 0.12.0

Reply:
I removed the bump